### ISSUE TITLE
Missing Mirana and Pudge predefined configuration

### ISSUE THREAD

**Issue by ahonn** [nervosnetwork/lumos#511]
I noticed that there are no configurations for Mirana and Pudge in https://github.com/ckb-js/lumos/blob/develop/packages/config-manager/src/predefined.ts. Only the Lina and Aggron4 configuration before the main network upgrade, maybe here we can change the name to the latest main network as well as the test network, or add two aliases.

**Comment by homura** [comment]
You're right, but considering the developer habits, and the documentation for Lumos that already exists in the community, the new aliases(Pudge/Mirana) may confuse the newcomers who are not familiar with CKB. 

Maybe we can keep this issue to check more feedback from developers?

If we eventually decide to make these new aliases available, Lumos team should contact some of the sites that have posted Lumos-related docs to push this update together
